deploy/helm/charts/templates/_helpers.tpl

```diff
@@ -220,7 +219,7 @@ Create labels for ndm cluster exporter component
 Create match labels for ndm node exporter deployment
 */}}
 {{- define "openebs-ndm.node-exporter.matchLabels" -}}
-app: {{ template "openebs-ndm.node-exporter.name" . }}
+app: {{ template "openebs-ndm.exporter.name" . }}
```

Comment:
What will be the value of {{ template "openebs-ndm.exporter.name" . }}?

Reply:
It will be `openebs-ndm-exporter`. So, basically the name is formed as chart-name + "exporter" and chart name is `openebs-ndm`.

Reply:
Ok. Saw this in the templates
{{- define "openebs-ndm.exporter.name" -}}
{{- $ndmName := default .Chart.Name .Values.ndmExporter.**nameOverride** | trunc 63 | trimSuffix "-" }}

Not able to locate **nameOverride** key in the values.yaml. How is that set?

Reply:
**nameOverride** is not present currently. It's just we have a option provisioned for it if some users wants to override the name.

Reply:
If the name is overridden, I think the match label will not work in service-monitor, since its hard-coded to "openebs-ndm-exporter" there.

Reply:
We can remove the nameOverride option in that case from the `openebs-ndm-exporter.name` template.

Reply:
Yes